docs/docs/getting-started/android-setup.md

```diff
@@ -1 +1,236 @@
-# Android Setup
\ No newline at end of file
+# Android Setup
+
+After you've completed the [Rust setup](docs/getting-started/rust-setup.md) you should be able to run `cargo run --bin android`. This will create a new folder called `MoproAndroidBindings`. It should look like the structure
+
+```sh
+MoproAndroidBindings
+├── jniLibs
+│   ├── arm64-v8a
+│   │   └── libuniffi_mopro.so
+│   ├── armeabi-v7a
+│   │   └── libuniffi_mopro.so
+│   ├── x86
+│   │   └── libuniffi_mopro.so
+│   └── x86_64
+│       └── libuniffi_mopro.so
+└── uniffi
+    └── mopro
+        └── mopro.kt
+```
+
+First we will create an android app through Android Studio. If you already have an app project, you can skip this step. We'll do File -> New -> New Project and create an Empty Activity.
+
+![create an android app project](/img/android-example-1.png)
```

Comment:
I think for this tutorial we should tell the user to set their view a certain way here: 
<img width="356" alt="image" src="https://github.com/zkmopro/mopro/assets/631020/b15dffb1-0cf9-42f8-bef4-07990954b242">


The folder layout changes if it's set to "Android" or "Project"